Makefile.devel

```diff
@@ -84,16 +84,17 @@ DEVELOPERS=\
 -u "gklein	Gerwin Klein	gerwin.klein@nicta.com.au" \
 -u "assia	Assia Mahboubi	assia.mahboubi@inria.fr" \
 -u "monnier	Stefan Monnier	monnier@iro.umontreal.ca" \
--u "tews	Hendrik Tews	hendrik@askra.de"
+-u "tews	Hendrik Tews	hendrik@askra.de" \
+-u "steck	Paul Steckler	steck@stecksoft.com"
 
 # PRERELEASE_PREFIX is used to match PRERELEASE_TAG in sed
 # line in tag target below, which edits $(DOWNLOADHTML)
 # The prereltag.txt is kept as a record in the distrib area
 # of the current pre-release version (currently not used explicitly
 # anywhere for web pages/whatever).
-PRERELEASE_PREFIX=4\.4\.1~pre
-PRERELEASE_TAG=4.4.1~pre
-### Formerly: PRERELEASE_TAG=4.4pre$(shell date "+%y%m%d")
+PRERELEASE_PREFIX=5\.0~pre
+PRERELEASE_TAG=5.0~pre
```

Comment:
Thank you @psteckler!
I had only mentioned README.md and proof-site.el in my comment, but a few other files should also be updated to 5.0-git : to sum up, could you update these as well?
https://github.com/psteckler/ProofGeneral/blob/server-protocol/Makefile.devel#L95-L96
https://github.com/psteckler/ProofGeneral/blob/server-protocol/etc/ProofGeneral.spec#L3
https://github.com/psteckler/ProofGeneral/blob/server-protocol/doc/ProofGeneral.texi#L60
https://github.com/psteckler/ProofGeneral/blob/server-protocol/doc/PG-adapting.texi#L54